src/constants/MarkdownFiles/posts/dmp-25-AmanChadha-week14.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "DMP '25 Week-14 Update: Refining Japanese Translations and Repository Workflow"
+excerpt: "This week: Improved Japanese translation accuracy based on mentor feedback, refined the translation prompt for better linguistic balance, and enhanced repository workflow for easier review."
+category: "DEVELOPER NEWS"
+date: "2025-09-25"
+slug: "2025-09-25-dmp-25-weekly-update-aman-chadha"
+author: "@/constants/MarkdownFiles/authors/aman-chadha.md"
+tags: "dmp25,sugarlabs,weeklyupdate,aman-chadha"
+image: "assets/Images/c4gt_DMP.webp"
+---
+
+# Weekly Update: Refining Japanese Translations and Repository Workflow
+
+Project: [JS Internationalization with AI Translation Support](https://github.com/sugarlabs/musicblocks/pull/4731)  
+Mentors: [Walter Bender](https://github.com/walterbender), [Devin Ulibarri](https://github.com/devinulibarri)  
+Week: September 18 – September 25, 2025  
+
+---
+
+## Mentor Feedback on Japanese Translations
+
+This week, I received valuable feedback from mentor **Devin Ulibarri** regarding the Japanese translations. He noted that many of the generated strings were **phonetic translations** rather than proper contextual ones.  

```

Comment:
Please remove the bold text here. It smells strongly of a machine-generated artifact.